### DISCUSSION TITLE
Replaygain plugin with ffmpeg plugin doesn't work.

### DISCUSSION THREAD

**Discussion by demerzell** [beetbox/beets#4524] [Q&A] [answered]
Hello everyone. 

I can't solve an issue with replaygain plugin. 

I have 18k flac files in my library and I just tried replaygain for the first time. Want to use ffmpeg because it's already installed because of the convert plugin. 

SO my issue is that replaygain sometimes works sometimes doesn't. I don't see any pattern. It keeps failing with the same files/albums. 
This is the output I'm getting: 
```

PS C:\Users\szzol> beet replaygain -aW music album
replaygain: Analyzing 1 albums ~ ffmpeg backend...
replaygain: analyzing Music album
replaygain: ffmpeg exited with status 1
Exception in thread Thread-12:
Traceback (most recent call last):
  File "C:\Users\szzol\AppData\Local\Programs\Python\Python310\lib\threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "C:\Users\szzol\AppData\Local\Programs\Python\Python310\lib\site-packages\beetsplug\replaygain.py", line 1088, in run
    raise exc[1].with_traceback(exc[2])
  File "C:\Users\szzol\AppData\Local\Programs\Python\Python310\lib\site-packages\beetsplug\replaygain.py", line 1346, in wfunc
    return func(*args, **kwargs)
  File "C:\Users\szzol\AppData\Local\Programs\Python\Python310\lib\site-packages\beetsplug\replaygain.py", line 1283, in <lambda>
    callback=lambda task: task.store(write)
AttributeError: 'NoneType' object has no attribute 'store'
```

I also tried beet replaygain -awf music album same result. 

I'm using 
```
beets version 1.6.1
Python version 3.10.7
plugins: albumtypes, convert, embedart, fetchart, inline, lastgenre, lyrics, mbsync, missing, replaygain, rewrite, unimported
```

So while I composed this post I was under the impression that I'm using **beets version 1.6.0** but to my surprise I'm on 1.6.1---

I don't know if this is related to the issue or not.

**Comment by demerzell** [comment, marked as answer]
Found the issue (unicode characters)
And the solution: https://beets.readthedocs.io/en/stable/reference/config.html?highlight=replace#asciify-paths
/thread

**Comment by sampsyo** [comment]
See also #4507 and #4506, which addressed the underlying problem.
